Dockerfile

```diff
@@ -25,8 +25,6 @@ COPY ./build/local_settings.py /code/semesterly/local_settings.py
 COPY ./build/run_parser.sh /code/run_parser.sh
 
 RUN pip3 install -r /code/requirements.txt
-# This is needed on newer ubuntu
-RUN pip3 install psycopg2-binary==2.8.6
```

Comment:
Did you manually validate locally that this doesn't break the build?

Reply:
Yes, this doesn't break it since I added it to the requirements.txt